abdiff/core/run_ab_transforms.py

```diff
@@ -101,22 +108,31 @@ def run_ab_transforms(
             )
             containers.append(container)
 
-    wait_for_containers(containers, timeout)
+    failed_containers = wait_for_containers(containers, timeout)
     logger.info(f"All {len(containers)} containers have exited.")
 
     log_file = aggregate_logs(run_directory, containers)
     logger.info(f"Log file created: {log_file}")
 
-    transformed_files = get_transformed_files(run_directory)
+    # errors from failed containers are accessible via aggregated logs
```

Comment:
I think this is a really nice decision.  By aggregating the logs from the containers _before_ raising an exception that something went wrong, we have an enduring artifact about what happened.  This could be invaluable if we are running 10, 20, 50 input files through.

Reply:
Agreed!

Reply:
Hooray! It'd be great if we could parse only the information we need (i.e., the error from the logs) or create a separate log file for errors, but I think this is sufficient for our first pass!